internal/runners/deploy/deploy.go

```diff
@@ -217,7 +218,12 @@ func configure(installpath string, envGetter runtime.EnvGetter, out output.Outpu
 
 	out.Notice(locale.Tr("deploy_configure_shell", sshell.Shell()))
 
-	fail := sshell.WriteUserEnv(env, sscommon.Deploy, userScope)
+	envData := sscommon.EnvData{
+		constants.RCAppendDeployStartLine,
+		constants.RCAppendDeployStopLine,
+		"user_env",
+	}
```

Comment:
The envData should still live in the file you had, so it's centralized and reusable. I was just saying the lookup stuff seemed excessive.